Answers.md

```diff
@@ -1,11 +1,19 @@
+Chad Jemmett
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+  React JS is a UI library to create versatile web components that can handle a lot of data.
+  React can build apps that can take lots of data and display it on the webpage quickly.
+  React solves the problem of websites getting lots of data.
 
 1.  What does it mean to _think_ in react?
+  Thinking in React means thinking about data going from the top to the bottom. Or from the outside in. Data starts at the top of the app with the state. Data from the state can be passed down through the components via props on the components. Thinking in React means thinking about creating components that do one thing, and do them well. You can add or remove a component without affecting the function of the rest of the app. It also means learning jsx.
 
 1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
+    A class component will usually handle the state of the app. It runs in the background moving data down through the components. A presentational component will handle the way the data looks on the screen. It takes the data from the parent class component and manipulates it to display it in a way that's pleasing to the user.
 
 1.  Describe state.
+    State is the heart of the React app. It moves the data around the app via the components. Props on the components move the data from state further down the tree. The components use the state data to display and change their look and behavior.
 
 1.  Describe props.
+  Props are the properties of the components. They take the data from the parent app and use it. Props are imutable. The components never modify the props. 

```

Comment:
Great job on all of these answers, especially your explanation on state vs props! 💯